src/main/webapp/app/applications/applications.module.ts

```diff
@@ -1,10 +1,10 @@
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { RouterModule } from '@angular/router';
+import {NgModule, CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
```

Comment:
formatting is wrong in all files. there should be space after { and before } here